### PR TITLE
update of libs, plus unit tests fix

### PR DESCRIPTION
**Pull request by elopezanaya**
# PR Details

### Description
_Include a description of the problem to be solved_

Updating libraries and unit tests fix, including adding a node script to copy files, since currently we use a `cp` command which fails if not being executed in a batch console.

## Solution Proposed

_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence

_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

image

image

### Copy script evidence

image
image

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

File: scripts/copy_files.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+
+// Define the source and destination file paths
+const sourceFile = path.join(__dirname, '../src/external/aria');
```
Comment (dleblondMSFT) [inline scripts/copy_files.js:5]: Shouldn't this be a responsability of the implementer, not the SDK?

Comment (elopezanaya) [reply scripts/copy_files.js:5]: this way we ensure telemetry is ready anywhere is consumed, by now the goal is just to simplify the build process, we could evaluate later if its needed

**Review by matan-hershberg-nuance — APPROVED** [review]

File: scripts/copy_files.js (hunk 1 above)
Comment (xTEddie) [inline scripts/copy_files.js:5]: I'd suggest to not use any OS platform specific path like "../a/b/c". 

You can use `path.join(path.dirname(__dirname), "src", "external", "aria")`

File: scripts/copy_files.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const destinationFile = path.join(__dirname, '../lib/external/aria');
```
Comment (xTEddie) [inline scripts/copy_files.js:6]: I'd suggest to rewrite this to be less platform specific:

`path.join(path.dirname(__dirname), "lib", "external", "aria")`

**Review by xTEddie — APPROVED** [review]
